DESCRIPTION

```diff
@@ -6,13 +6,17 @@ Authors@R: c(
   person("Matthew", "Lincoln", email = "matthew.d.lincoln@gmail.com", role = c("aut", "cre")),
   person("Louis", "Maddox", role = "ctb"))
 Description: Simple utility functions to read from and write to the Windows,
-  OS X, and X11 clipboards.
-Imports: utils
+    OS X, and X11 clipboards.
+Imports:
+    utils,
+    readr
```

Comment:
`readr` should go under `Suggests` instead of `Imports` - I don't want to add a hard dependency for a utility function. Make a note in the function documentation that `readr` would need to be installed.